multipaz-compose/build.gradle.kts

```diff
@@ -86,6 +87,7 @@ kotlin {
                 implementation(libs.androidx.camera.lifecycle)
                 implementation(libs.androidx.camera.view)
                 implementation(libs.androidx.lifecycle.extensions)
+                implementation(libs.face.detection)
```

Comment:
Will look in more detail later but this needs to be moved to a separate `multipaz-facedetect` library since it's a big dependency that not everyone will want. If you can do that split it right now it would be nice...